lms/urls.py

```diff
@@ -676,6 +677,9 @@
     ),
 ]
 
+# include all adg lms urls
```

Comment:
[Core change 1.1 ](https://github.com/OmnipreneurshipAcademy/edx-platform/wiki/Core-Changes#11-write-required-apis-for-application-page-2) is not moved here. I guess that core change is not valid, would please have a look and update the wiki accordingly.
@asadali145